File: vanhocvp_sources/P147PROH.cpp
#include<iostream>
#include<string>
 
using namespace std;
 
int main(){
	int t=1;
	while(1){
		
		string a,b;
		cin>>a>>b;
		if(a=="END"&&b=="END") return 0;
		cout<<"Case "<<t++<<": ";
		if(a.size()!=b.size()) cout<<"different"<<endl;
		else{
			int flag=1;
			for(int i=0;i<a.size();i++){
				for(int j=0;j<b.size();j++){
					if(a[i]==b[j]){
						b[j]=' ';
						break;
					}
					else if(j==b.size()-1&&a[i]!=b[j]){
						cout<<"different"<<endl;
						flag=0;
						break;
					}
				}
				if(flag==0) break;
			}
			if(flag==1) cout<<"same"<<endl;
		}
	}
} 

File: kha_banh_sources/P147PROH.py
test = 0
while True:
    test += 1 
    a = input()
    b = input()
    if a == b == "END":
        break
    a = sorted(a)
    b = sorted(b)
    print("Case {}: ".format(test) , end = "")
    print("same") if a == b else print("different")
 
